apps/integration/integration/containers/stack.py

```diff
@@ -0,0 +1,39 @@
+from testcontainers.compose import DockerCompose
+import pytest
+from integration import SYCAMORE_ROOT
+from typing import List
+
+
+def docker_compose(services: List[str] = []):
+    """
+    Get a docker compose object that controls some set of the sycamore stack
+    :param services: the list of services to control. [] -> all services
+    :return: the docker compose object
+    """
+    if len(services) > 0:
+        return DockerCompose(SYCAMORE_ROOT, services=services)
+    else:
+        return DockerCompose(SYCAMORE_ROOT)
+
+
+def set_version_tag(tag: str):
+    with open(SYCAMORE_ROOT / ".env", "r") as f:
+        lines = f.readlines()
+    lines[0] = f"VERSION={tag}\n"
```

Comment:
This seems remarkably delicate.  What if somebody puts a comment at the top of .env?

Reply:
well, don't! but also yeah will fix